app.py

```
@@ -0,0 +1,147 @@
+        tmp_path = f"./tmp_{time.time_ns()}.pdf"
+        with open(tmp_path, "wb") as f:
+            f.write(response.content)
+
+        loader = PyPDFLoader(tmp_path)
+        documents = loader.load()
+
+        splitter = RecursiveCharacterTextSplitter(
+            chunk_size=req.chunk_size,
+            chunk_overlap=50
+        )
+        split_docs = splitter.split_documents(documents)
```

Ruff (0.13.1)

Local variable `split_docs` is assigned to but never used

Remove assignment to unused variable `split_docs`

(F841)